server/acks_handlers.go

```diff
@@ -401,7 +401,9 @@ func generateRuleAckMap(acks []types.SystemWideRuleDisable) (ruleAcksMap map[typ
 		if err == nil {
 			ruleAcksMap[compositeRuleID] = true
 		} else {
-			log.Error().Err(err).Msgf(compositeRuleIDError, ack.RuleID, ack.ErrorKey)
+			log.Error().Err(err).Interface(ruleIDStr, ack.RuleID).
```

Comment:
does Str(ruleIDStr, string(ruleID)) not work in these?

Reply:
yes, but I preferred to use `Interface` in order to avoid converting types. It's just a random decision, not based on nothing major...